src/sentry/static/sentry/app/components/contextData.jsx

```diff
@@ -73,6 +72,39 @@ function analyzeStringForRepr(value) {
   return rv;
 }
 
+class ToggleWrap extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {toggled: false};
+  }
+
+  render() {
+    if (this.props.children.length === 0) {
+      return null;
+    }
+    if (this.highUp) {
+      return this.props.children;
+    }
+
+    const classes = ['val-toggle'];
+    if (this.state.toggled) {
+      classes.push('val-toggle-open');
+    }
+    return (
+      <span className={classes}>
```

Comment:
I didn't realize you could pass an array, does that work?

Reply:
probably not, oversight on my side.